packages/pigeon/lib/kotlin_generator.dart

```diff
@@ -543,6 +547,7 @@ class KotlinGenerator extends StructuredGenerator<KotlinOptions> {
                       });
                     });
                   } else {
+                    indent.writeln('var wrapped: List<Any?>');
```

Comment:
Fixes an unused variable warning (when we're in the `if` path, which doesn't use this) and an unnecessary initialization warning (since both the try and catch initialize it).